### PR TITLE
AST-1461 - W3 validation error for tabindex

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
Primary header menu W3 validation error showing for tabindex.

Remove the tabindex attribute from span and add it to anchor tag

### Screenshots
[]
- https://share.bsf.io/o0u72nd0

### Types of changes

Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
I have added a card.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/extras.php [hunk 1]
```diff
@@ -425,7 +424,7 @@ function astra_dropdown_icon_to_menu_link( $title, $item, $args, $depth ) {
 	}
 	foreach ( $item->classes as $value ) {
 		if ( 'menu-item-has-children' === $value ) {
-			$title = $title . '<span role="' . esc_attr( $role ) . '" class="dropdown-menu-toggle" tabindex="' . esc_attr( $tabindex ) . '" >' . $icon . '</span>';
+			$title = $title . '<span role="' . esc_attr( $role ) . '" class="dropdown-menu-toggle" >' . $icon . '</span>';
```
Comment (imnavanath) [inline inc/extras.php:427]: @GauravKhupse the issue will resolve here, by removing tabindex attr from span but ideally it should be assigned to `menu-link` list item. Can you please verify this once you can check other themes as well for ref.

Comment (GauravKhupse) [reply inc/extras.php:427]: Yes, I have seen anchor tag has to default focusable,refer :- https://share.bsf.io/kpu41B6A
Checked with blocksy theme.

Comment (imnavanath) [reply inc/extras.php:427]: Okay do get sync PR merged & add test cases in card.

**Review by imnavanath — APPROVED** [review]

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
+v3.8.0(Unreleased)
+- Fix: W3 validation - Removed An element with the attribute tabindex for dropdown-menu-toggle.
```
Comment (premanshup) [inline changelog.txt:2]:
```suggestion
- Fix: W3 validation - Removed an element with the attribute tabindex for dropdown menu toggle.
```

Comment (GauravKhupse) [reply changelog.txt]: Changes done!

**Pull request merged**